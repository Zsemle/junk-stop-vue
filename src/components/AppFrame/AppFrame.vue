<template>
  <div class="app-frame">
    <TheHeader/>
    <div>
      <StopOverview/>
    </div>
    <TheFooter/>
  </div>
</template>

<script lang="ts">
import Vue from 'vue';
import StopOverview from '../StopOverview/StopOverview.vue';
import TheHeader from '../TheHeader/TheHeader.vue';
import TheFooter from '../TheFooter/TheFooter.vue';

export default Vue.extend({
  name: 'AppFrame',
  components: {
    TheHeader,
    TheFooter,
    StopOverview,
  },
});
</script>

<style lang="scss" scoped>

</style>
