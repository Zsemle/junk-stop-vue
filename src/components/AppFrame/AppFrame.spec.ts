import { shallowMount } from '@vue/test-utils';
import AppFrame from './AppFrame.vue';

describe('AppFrame.vue', () => {
  it('renders without errors', () => {
    const wrapper = shallowMount(AppFrame);
    expect(wrapper).toMatchSnapshot();
  });
});
