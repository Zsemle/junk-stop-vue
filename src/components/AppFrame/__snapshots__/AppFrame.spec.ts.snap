// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFrame.vue renders without errors 1`] = `
<div class="app-frame">
  <theheader-stub></theheader-stub>
  <div>
    <stopoverview-stub></stopoverview-stub>
  </div>
  <thefooter-stub></thefooter-stub>
</div>
`;
